src/UI/Factory.php

```diff
@@ -336,4 +336,22 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A divider marks a thematic change in a sequence of other components.
+	 *   composition: >
+	 *     A divider consists of a horizontal ruler which may be accompanied by a label.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: >
+	 *       Dividers MUST be used in in container components that render a sequence of items.
```

Comment:
Is it a) MUST be used or b) MUST only be used? If a) I disagree. There can be dropdowns without dividers. If b) add only the clarify.